### DISCUSSION TITLE
Missing UserRepository.cs file

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6733] [General]
Hi,

in Multi-tenancy tutorial, Filtering Users By TenantId paragraph, we talk about UserRepository.cs

but I'm not able to find that file.

I've installed Serene 6.5.1 on Visual Studio 2022.

Any idea?

Thank you, 
Elena.

**Comment by wezmag** [comment]
`UserRepository.cs` does not exist anymore in the new version.
Modify `UserListHandler.cs` in `RequestHandlers` folder instead.

**Reply by elenaaralla** [reply]
> `UserRepository.cs` does not exist anymore in the new version. Modify `UserListHandler.cs` in `RequestHandlers` folder instead.

ok, thank you... 
so tutorial at https://serenity.is/docs/tutorials/multi_tenancy/filtering_users_by_tenantid is obsolete?
Is there an updated version?
Thank you, 
Elena
